build_component.py

```diff
@@ -295,8 +295,14 @@ def main():
   os.chdir(os.path.join(repos_root_dir, ".."))
   # Copy the necessary files to the respective target folders, 
   # following the instructions in scripts/config_build.txt.
-  config_file = open("scripts/config_build.txt")
-  
+  try:
+    config_file = open("scripts/config_build.txt")
+  except IOError:
+    print "unable to open file scripts/config_build.txt ! This \
+    script is not meant to be called individually, but through \
+    a wrapper script, build.py. See the Seattle build instructions \
+    wiki for details: https://seattle.poly.edu/wiki/BuildInstructions"
+    return 0
```

Comment:
The official Python tutorial has a section on [exceptions](https://docs.python.org/2/tutorial/errors.html#raising-exceptions).  One approach you may follow here is to log/print information about the required`scripts/config_build.txt` file, and then re-raise the exception with a bare `raise` (to avoid masking the original exception). 

The following article explains why exceptions can lead to cleaner code:
[Write Cleaner Python: Use Exceptions](http://www.jeffknupp.com/blog/2013/02/06/write-cleaner-python-use-exceptions/)